Frameworks/UIKit/NSMutableParagraphStyle.mm

```diff
@@ -21,19 +21,32 @@
 @implementation NSMutableParagraphStyle
 
 /**
- @Status Stub
+ @Status Interoperable
 */
 - (id)copyWithZone:(NSZone*)zone {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return [self mutableCopyWithZone:zone];
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 - (void)setParagraphStyle:(NSParagraphStyle*)obj {
```

Comment:
wut?

---
In reply to: [87270987](https://github.com/Microsoft/WinObjC/pull/1343#discussion_r87270987) [](ancestors = 87270987)